src/DataGrid.tsx

```diff
@@ -82,7 +82,7 @@ interface EditCellState<R> extends Position {
 
 type DefaultColumnOptions<R, SR> = Pick<
   Column<R, SR>,
-  'renderCell' | 'width' | 'minWidth' | 'maxWidth' | 'resizable' | 'sortable'
+  'renderCell' | 'width' | 'minWidth' | 'maxWidth' | 'resizable' | 'sortable' | 'draggable'
```

Comment:
If the column is both frozen and draggable then I think users can decide how to order them. RDG will still sort frozen columns first